### PR TITLE
Fix #294 string(REPLACE ...) fails if _PKG_CONFIG_REQUIRES is empty.

### LINKED ISSUE #294: package-config.cmake string(replace ...) fails if _PKG_CONFIG_REQUIRES is empty
In https://github.com/jrl-umi3218/jrl-cmakemodules/blob/7eca9ee6c9d1c4ee20eb82272e94f9d11642053a/package-config.cmake#L126
if _PKG_CONFIG_REQUIRES is empty this line fails.

### PR DESCRIPTION
**Pull request by olivier-stasse**

### CODE REVIEW

File: package-config.cmake [hunk 1]
```diff
@@ -123,7 +123,9 @@ write_basic_package_version_file(
 #   * TARGETS_EXPORT_NAME
 #   * PROJECT_NAME
 #   * _PKG_CONFIG_REQUIRES_LIST
-string(REPLACE "," ";" _PKG_CONFIG_REQUIRES_LIST ${_PKG_CONFIG_REQUIRES})
```
Comment (gergondet) [inline package-config.cmake]: I think a more robust patch is
```cmake
string(REPLACE "," ";" _PKG_CONFIG_REQUIRES_LIST "${_PKG_CONFIG_REQUIRES}")
```

**Comment by olivier-stasse** [comment]
Done.

**Comment by olivier-stasse** [comment]
Thanks for the quick feedback

**Comment by gergondet** [comment]
No problem, merging now.

**Pull request merged**